jsonrpc/client.go

```diff
@@ -53,37 +53,75 @@ func (sc *Client) AccountNonce(context.Context, string) (*big.Int, error) {
 	panic("not implemented")
 }
 
+func (sc *Client) BlockNumber(ctx context.Context) (*big.Int, error) {
+	var blockNumber big.Int
+	err := sc.c.CallContext(ctx, &blockNumber, "starknet_blockNumber")
+	if err != nil {
+		return nil, err
+	}
+	return &blockNumber, err
+}
+
+func (sc *Client) LatestBlock(ctx context.Context, scope string) (*types.Block, error) {
```

Comment:
Hmm I think the library should only implement the endpoints defined by the spec and abstractions should be left to the consumer.